pkg/arvo/app/chat-hook.hoon

```diff
@@ -644,6 +654,20 @@
     ==
   ==
 ::
+++  poke-import
+  |=  jammed=@
+  ^-  (quip card _state)
+  =/  sty=state-8  ;;(state-8 (cue jammed))
```

Comment:
Also needs to be upgraded for current state number

Reply:
Why isn't this just `versioned-state` or w/e?